main.tf

```diff
@@ -42,56 +41,76 @@ module "project-services" {
     "sql-component.googleapis.com",
     "sqladmin.googleapis.com",
     "storage.googleapis.com",
-    "secretmanager.googleapis.com",
     "run.googleapis.com",
     "redis.googleapis.com",
   ]
 }
 
 resource "google_service_account" "runsa" {
   project      = var.project_id
-  account_id   = "${var.deployment_name}-run-sa"
```

Comment:
Any reason for removing the -run-sa prefix? I think it helps to understand it's the SA used by CR.